### DISCUSSION TITLE
velero in okd error="open /tmp/plugin3747443965: read-only file system"

### DISCUSSION THREAD

**Discussion by sdarvell** [vmware-tanzu/velero#4924] [Community support Q&A]
Whenever I install velero in okd, either through something like this
https://velero.io/docs/v1.8/contributions/minio/
or the oadp operator in okd, I get this error message in the logs

It doesn't seem to matter what I use as plugin settings, or what plugin it uses (i'm trying to use the aws pone, I can't get passed this. Thought it was that I had 2 different storage classes installed and maybe that was confusing things. Deleted (openebs) then tried, then deleted the other and tried, all attempts end up with the below errror message.

Is there something i'm doing wrong?

time="2022-05-18T13:27:42Z" level=info msg="setting log-level to INFO" logSource="pkg/cmd/server/server.go:175"
time="2022-05-18T13:27:42Z" level=info msg="Starting Velero server konveyor-dev (-)" logSource="pkg/cmd/server/server.go:177"
time="2022-05-18T13:27:42Z" level=info msg="1 feature flags enabled [EnableCSI]" logSource="pkg/cmd/server/server.go:179"
time="2022-05-18T13:27:42Z" level=error msg="plugin init error" cmd=/velero error="open /tmp/plugin3747443965: read-only file system" logSource="pkg/plugin/clientmgmt/logrus_adapter.go:89" pluginName=velero
An error occurred: Unrecognized remote plugin message:

This usually means that the plugin is either invalid or simply
needs to be recompiled to support the latest protocol.

**Comment by berendiwema** [comment]
I've seen similar issues on OpenShift 4 clusters where restrictive SCC's were defined with a too high priority.
